xtask/src/main.rs

```diff
@@ -40,6 +43,16 @@ fn try_main() -> Result<()> {
             let task = ReleaseTask::new(cmd.name, project_root)?;
             task.run()
         }
+        flags::XtaskCmd::Ci(ci) => {
+            println!(
+                "CI Tests are running on {} using {}...",
+                ci.crates.as_ref().unwrap_or(&"all".to_string()),
+                ci.version.as_ref().unwrap_or(&"all".to_string()),
```

Comment:
You can use `.as_deref` to you don't need the `.to_string()` for the string literals.